index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Landing Page</title>
+<meta charset="UTF-8">
+<link rel="stylesheet" href="style.css">
+</head>
+<body>
+<header id="header">
+    <img id = "header-img" src = "https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.istockphoto.com%2Fphotos%2Flone-wolf&psig=AOvVaw262V3pwgOca0kyLRc0_dWN&ust=1645095402108000&source=images&cd=vfe&ved=0CAsQjRxqFwoTCKCH-4yIhPYCFQAAAAAdAAAAABAD" alt = "logo" class="logo"/>
```

Comment:
Kindly check this image as its not displaying on my browser

![image](https://user-images.githubusercontent.com/91391736/170953281-fc20f3e3-d3b9-426b-87c2-c64f2157430d.png)